desktop.css

```diff
@@ -0,0 +1,160 @@
+@media (min-width: 768px) {
+  .mobile-visible {
+    display: none;
+  }
+
+  .header {
+    padding: 1% 10%;
+    width: 80%;
+  }
+
+  .portfolio {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    gap: 24px;
+    color: #344563;
+    padding: 10px 12px;
+  }
+
+  .first-page {
+    padding: 20% 18%;
+    background-image: url('images/desktop-version/main-page-bg.svg');
+  }
+
+  .myheader {
+    font-size: 48px;
+    line-height: 60px;
+    color: #091e42;
+    letter-spacing: 0.37px;
+  }
+
+  .projects-section {
+    padding: 10%;
+    gap: 142px;
+  }
+
+  .project {
+    grid-template-columns: 1fr 1fr;
+    border-radius: 24px;
+    gap: 24px;
+  }
+
+  .project-title {
+    font-size: 40px;
+  }
+
+  .project-client {
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .project-meta-data {
+    font-size: 24px;
+    margin-top: 20px;
+  }
+
+  .project-image-desk {
+    height: 100%;
+    width: 100%;
+  }
+
+  .project-heading {
+    padding: 24px;
+    gap: 24px;
+  }
+
+  .skill-title > p {
+    margin-top: 24px;
+    margin-left: 24px;
+    margin-bottom: 24px;
+  }
+
+  .skill-title > img {
+    margin-right: 24px;
+  }
+
+  .project:nth-child(odd) .project-image-container {
+    order: 1;
+  }
+
+  .project:nth-child(1) img {
+    content: url(images/desktop-version/1.png);
+  }
+
+  .project:nth-child(2) img {
+    content: url(images/desktop-version/2.png);
+  }
+
+  .project:nth-child(3) img {
+    content: url(images/desktop-version/3.png);
+  }
+
+  .project:nth-child(4) img {
+    content: url(images/desktop-version/4.png);
+  }
+
+  .about-section {
+    grid-template-columns: 40% 1fr;
+    padding: 10%;
+    gap: 25px;
+  }
+
+  .content {
+    font-size: 1rem;
+    line-height: 28px;
+    color: #344563;
+    width: 97%;
+  }
+
+  .languages {
+    display: flex;
+    flex-direction: row;
+    gap: 20px;
+  }
+
+  .img-background {
+    padding: 20px;
+    width: 122px;
+    height: 120px;
+    background-color: #f7f7f9;
+  }
+
+  .skill-title {
+    display: flex;
+    justify-content: space-between;
+    font-size: 20px;
+    gap: 20px;
+  }
+
+  .contact-me {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    color: #ebebff;
+    padding: 24px;
+    width: 569px;
+    margin: 0 auto;
+    margin-top: 50px;
+  }
+
+  footer {
+    background: #6070ff url(images/desktop-version/contact-form-bg.svg);
+    background-position-x: right;
+    background-repeat: no-repeat;
+    border-top-left-radius: 100px;
+  }
+
+  .get-in-touch button {
+    color: #6070ff;
+    font-weight: 500;
+    font-size: 17px;
+    background: #fff;
+    padding: 12px;
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+  }
+}
```

Comment:
- [ ] I've spotted a content overlapping issue, consider taking a look at it and fixing it before your next review.

![FireShot Capture 009 - HTML 5 My Portfolio - raw githack com](https://user-images.githubusercontent.com/52143815/177975634-0c748400-b258-45a1-ba5f-8c574e3b0bd4.png)